shell/platform/linux/fl_text_input_plugin.cc

```diff
@@ -175,6 +181,19 @@ static FlMethodResponse* set_client(FlTextInputPlugin* self, FlValue* args) {
     self->input_action = g_strdup(fl_value_get_string(input_action_value));
   }
 
+  // clear multiline state and set again only if this current field is multiline
```

Comment:
Comments should be sentences, with proper capitalization and punctuation (i.e., ending with "."), per the style guide.

Reply:
sorry about that, fixed now.